configs/benchmark/pytorch/pytorchxla_torchbench_config.py

```diff
@@ -30,40 +30,28 @@ def set_up_torchbench(model_name: str = "") -> Tuple[str]:
       "sudo apt install -y libopenblas-base",
       "sudo apt install -y libsndfile-dev",
       "sudo apt-get install libgl1 -y",
+      "sudo chmod 777 /usr/local/lib/python3.10/dist-packages/",
```

Comment:
Yes, that should also work. I can update and do the test.

One concern for using `--user` is that the installation path is not in the default `$PATH`, and this can cause the installation complain.